akka-persistence-typed/src/test/scala/docs/akka/persistence/typed/BasicPersistentBehaviorsCompileOnly.scala

```diff
@@ -50,10 +63,14 @@ object BasicPersistentBehaviorsCompileOnly {
   val samplePersistentBehavior = PersistentBehaviors.receive[Command, Event, State](
     persistenceId = "abc",
     emptyState = State(),
-    commandHandler = (ctx, state, cmd) ⇒ ???,
-    eventHandler = (state, evt) ⇒ ???)
-    .onRecoveryCompleted { (ctx, state) ⇒
-      ???
+    commandHandler =
+      (ctx, state, cmd) ⇒
+        throw new RuntimeException("TODO: process the command & return an Effect"),
+    eventHandler =
+      (state, evt) ⇒
+        throw new RuntimeException("TODO: process the event return the next state")
+  ).onRecoveryCompleted { (ctx, state) ⇒
+      throw new RuntimeException("TODO: add some end-of-recovery side-effect here")
```

Comment:
add the same `??? // TODO: ` for the Scala examples?

Reply:
sorry, this was the Scala examples, that's all good

Reply:
No, I decided to put the documentation in the exception instead, so if you forget, you'll know.